content/en/about-the-docs/_index.md

```diff
@@ -1,37 +1,31 @@
 ---
 title: "About the Docs"
-date: 2023-04-21
-version: 'Corda 5.0'
-menu:
-  corda5:
-    identifier: corda5-about-the-docs
-    parent: corda5-intro
-    weight: 1000
-section_menu: corda5
+date: 2023-08-16
 ---
 # About the Docs
 The following topics are explained on this page:
-* [Documenting Corda]({{< relref "#documenting-corda">}})
-* [End of Life Strategy]({{< relref "#end-of-life-strategy">}})
 * [Reporting an Error in the Docs]({{< relref "#reporting-an-error-in-the-docs">}})
+* [Documentation End of Life Strategy]({{< relref "#documentation-end-of-life-strategy">}})
+* [Documenting Corda 5]({{< relref "#documenting-corda-5">}})
 
-## Documenting Corda
-The Corda 5 documentation is structured differently than in previous versions of Corda. Previously, Corda Enterprise and Corda Community content was published in separate documentation sets. 
-From Corda 5.0 onwards, there is one documentation set, but the features that relate to Corda Enterprise are marked to avoid confusion. 
+## Reporting an Error in the Docs
+The R3 Technical Writing team continuously strive to improve our documentation and overall content strategy. As we make improvements, we would love to get your feedback on what we are doing well, and what we could be doing better. 
 
-{{< 
-  figure
-	 src="docs-enterprise-icon-use-case.png"
-   width=55%
-	 figcaption="Documentation heading with Enterprise icon"
-	 alt="Enterprise icon"
->}}
+Click [here](mailto:docs@r3.com) to email the R3 Technical Writing team. To help us deal with your feedback effectively, please include all of the following:
 
-* **A**: The heading without the 'Enterprise only' icon is the most common and it is used where the content applies to all Corda users.
-* **B**: The ‘Enterprise only' icon is used mostly in headings, indicating that the content under that heading relates to a Corda Enterprise feature. On occasions, the icon may be used inline to identify sentences, paragraphs, or tables that apply to a Corda Enterprise feature. 
+* Your name
+* Your company name
+* Your email address (if different from the one being used to report)
+* The URL of the documentation page (if it is an issue being reported)
+
+Please include as much detail as you can in your feedback, as it is valuable to us and it will be read and shared with the rest of the team. A ticket is then created for any issues received, which will be quickly triaged and if necessary, assigned to a Technical Writer to effect changes.
```

Comment:
```suggestion
Please include as much detail as you can in your feedback, as it is valuable to us and it will be read and shared with the rest of the team. A ticket is then created for any issues received, which will be quickly triaged and, if necessary, assigned to a Technical Writer to effect changes.
```